website/src/static/index.html

```diff
@@ -6,9 +6,9 @@
     <meta name="description" content="Kepler.gl is a data agnostic, WebGL empowered, high-performance web application for geospatial analytic visualizations.">
     <title>kepler.gl</title>
     <link rel="shortcut icon" href="/favicon.png">
-    <link rel="stylesheet" href="//d1a3f4spazzrp4.cloudfront.net/kepler.gl/uber-fonts/4.0.0/superfine.css">
+    <link rel="stylesheet" href="https://d1a3f4spazzrp4.cloudfront.net/kepler.gl/uber-fonts/4.0.0/superfine.css">
 
-    <link href="//api.tiles.mapbox.com/mapbox-gl-js/v0.42.0/mapbox-gl.css" rel="stylesheet">
+    <link href="https://api.tiles.mapbox.com/mapbox-gl-js/v0.42.0/mapbox-gl.css" rel="stylesheet">
```

Comment:
this was left out intentionally, cause github will figure out which protocol to use. It doesn't cause any performance issues for windows, as far as I know. Why remove it?

Reply:
In this particular case it won't case an issue because this file is used for our website and not to export the map but I do agree we should be consistent so I will update this one

Reply:
I did this to be consistent with other html files.